mypy/plugin.py

```diff
@@ -104,7 +104,7 @@
    another module, such as the builtins, that is outside the current module or
    import cycle, you can safely assume that you won't receive a placeholder.
 
-When testing your plugin with the new semantic analyzer, you should have a test
+When testing your plugin with the semantic analyzer, you should have a test
```

Comment:
TBH this sound a bit awkward. I would just say it is recommended to have a test like below.

```suggestion
When testing your plugin, you should have a test
```